symphony/bdk/core/service/message/multi_attachments_messages_api.py

```diff
@@ -0,0 +1,214 @@
+from symphony.bdk.gen.agent_api.messages_api import MessagesApi
+from symphony.bdk.gen.agent_model.v4_message import V4Message
+from symphony.bdk.gen.agent_model.v4_message_blast_response import V4MessageBlastResponse
+from symphony.bdk.gen.api_client import Endpoint
+from symphony.bdk.gen.model_utils import (
+    file_type
+)
+
+
+class MultiAttachmentsMessagesApi(MessagesApi):
+
+    def __init__(self, api_client=None):
+        super().__init__(api_client)
+
+        def __v4_message_create_post(self, **kwargs):
+            kwargs['async_req'] = kwargs.get(
+                'async_req', False
+            )
+            kwargs['_return_http_data_only'] = kwargs.get(
+                '_return_http_data_only', True
+            )
+            kwargs['_preload_content'] = kwargs.get(
+                '_preload_content', True
+            )
+            kwargs['_request_timeout'] = kwargs.get(
+                '_request_timeout', None
+            )
+            kwargs['_check_input_type'] = kwargs.get(
+                '_check_input_type', True
+            )
+            kwargs['_check_return_type'] = kwargs.get(
+                '_check_return_type', True
+            )
+            kwargs['_host_index'] = kwargs.get('_host_index')
```

Comment:
Please mind that it only puts a None value for key `_host_index` if it key was not existing before. Is this needed?

Reply:
same answer as the comment above

Reply:
👌